routes/users.js

```diff
@@ -215,6 +217,113 @@ router.get("/:userId/offerings/inactive", async (req, res) => {
   }
 });
 
+// Get all drilled down details about a postings by the user of the given userId (Use this to get all the offerings pending)
+router.get("/:userId/postings/complete", async (req, res) => {
+  try {
+    const postingsOfUser = await Posting.find({
+      ownerId: req.params.userId,
+      active: true,
+    }).populate("offerings");
+
+    res.status(200).json(postingsOfUser);
+  } catch (err) {
+    res.status(500).json({
+      message: "Error code 500: Failed to process request",
+    });
+  }
+});
+
+// Add review to the user with userName
+router.post("/:userName/reviews", async (req, res) => {
+  const review = await new Review({
+    title: req.body.title,
+    review: req.body.review,
+    rating: req.body.rating,
+    reviewUsername: req.body.reviewUsername,
+  }).save();
+
+  try {
+    const userToUpdate = await User.findOneAndUpdate(
+      {
+        userName: req.params.userName,
+      },
+      { $push: { reviews: review._id } },
+      { new: true }
+    );
+
+    res.status(201).json(userToUpdate);
```

Comment:
Oh to be honest I didn't give much thought to what to return. Now thinking about it, should we bother returning anything at all for POST endpoints? Perhaps a simple status code is sufficient? Id imagine this would help with the performance too unless the frontend would want to render the updated content from the response? Let me know what you folks think.